### DISCUSSION TITLE
Deployment does not have minimum availability.

### DISCUSSION THREAD

**Discussion by CROprogrammer** [strimzi/strimzi-kafka-operator#7947] [Q&A]
I have a situation where my Strimzi operator keeps failing and restarting with message 
`installing: waiting for deployment strimzi-cluster-operator-v0.32.0 to become ready: deployment "strimzi-cluster-operator-v0.32.0" not available: Deployment does not have minimum availability.`
What could it be?

**Comment by scholzj** [comment]
Maybe a start would be to explain _where_ are you getting this message.

**Reply by CROprogrammer** [reply]
I have installed Strimzi operator 0.32.0 on Openshift platform.
![image](https://user-images.githubusercontent.com/73464772/213662472-40bb4f12-2407-4a5b-a465-8ac14270a33b.png)

Please tell me what else do you need to know.

**Reply by scholzj** [reply]
I guess you would need to check the deployment created by OPenShift when installing it. But it says _Install succeeded_, so it is not really clear to me if there is any problem.

**Reply by CROprogrammer** [reply]
So its not a problem that its keeps happeding from time to time?

**Reply by scholzj** [reply]
Well, I do not really know what is happening. So I do not know if it is or isn't a problem I'm afraid.
